test/test_matmul_cuda.py

```diff
@@ -1287,14 +1293,21 @@ def test_grouped_gemm_2d_2d(self, fast_accum, strided):
             ascalelist.append(scale_a[i * m : (i + 1) * m])
             bscalelist.append(scale_b[i * n : (i + 1) * n])
             start = offs_cpu[i]
+        print(a)
+        print("alist:", alist)
+        asplit = [x.squeeze(1) for x in a.view(m, n_groups, k).split(1, dim=1)]
+        print("asplit:", asplit)
```

Comment:
stray print?

Reply:
whoops, need to be more careful with my debug code 😬